src/JavascriptVM/vm.ts

```diff
@@ -233,7 +233,12 @@ export class BlocklyInterpreter {
       );
     });
 
+    // this may cause an infinite loop, we should bail out if this is the case
+    this.interpreter.run();
+
     // Start running in a paused state - ie, spawn a "thread"
+    this.interpreter.appendCode("start();");
```

Comment:
If someone 'load's code that does not define a `start()` function, then nothing happens. Should we instead maybe emit a warning?  I am aware that implementing such a check in a way that works in all possible scenarios might be tricky (or an unreasonable amount of effort) - but given that we control the code that gets emitted for a start block we might be able to come up with a check that should work in most (almost all?) cases?

Reply:
I'm not sure how to start this one. I think we can wait till we've seen this happen first.